### PR TITLE
update reward ceiling, change eligibility from 60 to 30 days

### PR DESCRIPTION
**Pull request by jennypollack**
https://gov.centrifuge.io/t/request-for-comments-radial-rewards-parameter-update/1769

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -2,15 +2,15 @@ import { dataSource, BigInt } from '@graphprotocol/graph-ts'
 
 export const secondsInDay = 86400
 
-export const secondsInSixtyDays = 5184000
+export const secondsInThirtyDays = 259200
```
Comment (philipstanislaus) [inline src/config.ts:5]:
```suggestion
export const secondsInThirtyDays = 2592000
```

Comment (philipstanislaus) [reply src/config.ts:5]: Or, even better: 

```suggestion
export const secondsInThirtyDays = 30 * secondsInDay
```

**Review by philipstanislaus — COMMENTED** [review]
Do we really want to change the 60 days to 30 days also for the past?

CC @lucasvo

**Comment by jennypollack** [comment]
going off of lucas' comment in slack: 
"I think we should just change the 60 to 30 days and make it retroactive. I don't care if we award a bit too much. WDYT?"

**Review by philipstanislaus — APPROVED** [review]

**Comment by philipstanislaus** [comment]
Yes, the only concern would be that this is cross-chain and the balances will change. But if I'm not mistaken, it would only increase rewards, if anything, so that should be fine.

**Pull request merged**